kzg_prover/src/circuits/utils.rs

```diff
@@ -91,6 +91,7 @@ pub fn full_prover<C: Circuit<Fp>>(
     AdviceSingle<halo2_proofs::halo2curves::bn256::G1Affine, Coeff>,
     Fp,
 ) {
+    #[cfg(feature = "profiling")]
```

Comment:
I added this feature flag for hiding these logs while benchmark testing, it's like one below:
```
Benchmarking <K = 17, N_USER = 16, N_CURRENCIES = 2> range check: Collecting 10 samples in estimated 81.192 s (10 iterations)
Start:   Creating proof
End:     Creating proof ............................................................8.086s
... 
```
We can check the results of benchmark instead of that.